docker/init.sh

```diff
@@ -42,6 +42,9 @@ fi
 echo ">>> Upgrading database..."
 bundle exec rake db:migrate
 
+echo ">>> Seed database..."
+bundle exec rake db:seed
```

Comment:
This should happen after the db:schema:load and inside the [if statement](https://github.com/openSUSE/osem/blob/master/docker/init.sh#L37-L40), so that it only runs once.

Another way to do this, is to replace db:schema:load with db:setup